src/locale/localed-util.c

```diff
@@ -86,6 +164,19 @@ void context_clear(Context *c) {
         c->polkit_registry = bus_verify_polkit_async_registry_free(c->polkit_registry);
 };
 
+static X11Context *context_get_x11_context(Context *c) {
+        assert(c);
+
+        if (!x11_context_isempty(&c->x11_from_xorg))
+                return &c->x11_from_xorg;
+
+        return NULL;
+}
```

Comment:
That's also for this function. See the comment below. The function will be extended in a later commit, and cannot make this 'const', as it will be used in `_safe()`.